src/Spryker/Zed/ProductMerchantPortalGui/Communication/Form/ProductLocalizedAttributesForm.php

```diff
@@ -139,6 +139,8 @@ protected function addNameField(FormBuilderInterface $builder)
                     'max' => 255,
                 ]),
             ],
+            'sanitize_xss' => true,
```

Comment:
this functionality is available only in GUI 3.48

Reply:
2.15.0 version has 3.48 GUI, so all good

Reply:
I see
Roman introduced another backport recently